app/services/data_migrations/backfill_enic_reason.rb

```diff
@@ -0,0 +1,16 @@
+module DataMigrations
+  class BackfillEnicReason
+    TIMESTAMP = 20240729111251
+    MANUAL_RUN = false
+
+    def change
+      ApplicationQualification.where(enic_reason: nil).where.not(enic_reference: nil).in_batches do |batch|
+        batch.update_all(enic_reason: 'obtained')
+      end
+
+      ApplicationQualification.where(enic_reference: nil).in_batches do |batch|
```

Comment:
Should uk qualifications be 'maybe' or 'not needed'? Or has that been done already?